### DISCUSSION TITLE
Issues with getting the server to run

### DISCUSSION THREAD

**Discussion by Darkblader24** [wolveix/satisfactory-server#105] [Q&A] [answered]
Hey, I'm trying to get this docker version of the Satisfactory server running but it just doesn't work. I have limited experience with docker so far. 

I want to use port 8888 instead of 7777 because of Terraria using the same port. So here is how I started the image:

`sudo docker run -d --name=satisfactory-server -h satisfactory-server -e MAXPLAYERS=8 -e PGID=1000 -e PUID=1000 -e SERVERGAMEPORT=8888 -e STEAMBETA=false -v /home/philipp/Servers/Satisfactory:/config -p 8888:8888/udp -p 15000:15000/udp -p 15777:15777/udp --restart unless-stopped wolveix/satisfactory-server:latest`

Here is the log: https://pastebin.com/WBs3geKH

I can't connect to the server ingame. I'm not sure what goes wrong here, but I think it's stuck in this state. Any help is appreciated, thank you!

Edit: Also which of these 3 ports is the one you use to actually connect ingame?

**Comment by wolveix** [comment, marked as answer]
Hey. Please open this as an issue in the future, as discussions are not intended for this purpose.

The log shows that the server is online and fine, so the issue is certainly a networking one. How are you trying to connect to it in-game? Are you supplying the IP and port?

**Comment by Darkblader24** [comment]
Turns out that my router forwarded the port to my PC instead of to my server even though I don't block any ports in the router. My guess is that Satisfactory on my PC snatched that port from my router when I first started it. Now I forward the port directly to the server and it works. Thank you for the help!
